releasenotes/releasenotes_develop.md

```diff
@@ -1,52 +1,299 @@
-# Release Notes develop
+# Release Notes 0.8.0
 
-...
+Keptn 0.8 improves the core use cases of continuous delivery and automated operations by implementing the new Shipyard version [v0.2](https://github.com/keptn/spec/tree/0.2.0). This new Shipard version has been proposed and refined in [KEP 06](https://github.com/keptn/enhancement-proposals/pull/6).
 
-**The key announcements of Keptn x.x.x**:
+---
 
+**The key announcements:**
```

Comment:
Should this be just "Key announcements"?

Reply:
true, thx. Fixed it.